src/pages/nft/claim/index.vue

```diff
@@ -729,13 +729,19 @@ export default {
     },
     loginAddress(newValue) {
       this.claimingAddress = newValue;
+      if (this.claimingAddress && this.state === NFT_CLAIM_STATE.LOGIN) {
+        this.state = NFT_CLAIM_STATE.ID_CONFIRMATION;
+      }
       if (!this.getAddress && !(this.status === 'completed')) {
         this.claimingAddress = '';
       }
```

Comment:
Would the `this.getAddress` logic below affect how we judge if `this.claimingAddress` should trigger state change?

Reply:
Removed, since the loginAddress is stored in session storage, we can just determine if a user is logged out by checking getAddress.